app/services/ingest_map_set_service.rb

```diff
@@ -0,0 +1,105 @@
+class IngestMapSetService < IngestScannedMapsService
+  attr_reader :record, :base_file_path, :user
+
+  def ingest_map_set(record, base_file_path, user)
+    @record = record
+    @base_file_path = base_file_path
+    @user = user
+
+    delete_duplicates!("source_metadata_identifier_tesim:\"#{RSolr.solr_escape(record['bibid'])}\"") if record['bibid']
+    map_set = create_map_set
+    members = create_map_set_members
+    map_set.ordered_members = members
+    map_set.save!
+  end
+
+  def minimal_record(klass, user, attributes)
+    default_attributes = { rights_statement: ['http://rightsstatements.org/vocab/NKC/1.0/'],
+                           visibility: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC }
+    r = klass.new
+    r.attributes = default_attributes.merge(attributes)
+    r.apply_depositor_metadata user
+    r.apply_remote_metadata if r.source_metadata_identifier
+    r.id = ActiveFedora::Noid::Service.new.mint
+    r.save!
+    Workflow::InitializeState.call(r, workflow_name, workflow_state(klass))
+    @logger.info "Created #{klass}: #{r.id} #{attributes}"
+
+    r
+  end
+
+  def workflow_state(klass)
+    klass == MapSet ? 'final_review' : 'complete'
+  end
+
+  def ingest_work(file_path, ark, title, sheet_title)
+    klass = ImageWork
+    bib_id = record['bibid']
+    attribs = { source_metadata_identifier: [bib_id], identifier: ["ark:/88435/#{ark}"] }
+    r = minimal_record klass, user, attribs
+    r.source_metadata_identifier = []
```

Comment:
Because it's not really the source metadata for that record, we're really just pulling from the parent MapSet and then changing the title. And bounding box (one day). If someone refreshed all the image works from bibdata, it would overwrite the correct title.